cli/tests/unit_node/_fs/_fs_dir_test.ts

```diff
@@ -0,0 +1,209 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import {
+  assert,
+  assertEquals,
+  fail,
+} from "../../../../test_util/std/testing/asserts.ts";
+import { assertCallbackErrorUncaught } from "../_test_utils.ts";
+import Dir from "ext:deno_node/_fs/_fs_dir.ts";
+import type Dirent from "ext:deno_node/_fs/_fs_dirent.ts";
```

Comment:
`ext:` only works for the internal Deno code. I think these should be imported like the below:

```suggestion
import { Dir, type Dirent } from "node:fs";
```

Reply:
Thanks for helping out, I have made the changes according to your suggestion.
Please have a look.